apps/condo/domains/ticket/utils/serverSchema/resolveHelpers.js

```diff
@@ -282,8 +285,46 @@ async function classifyTicket (context, details) {
     })
 }
 
+const STATUS_RELATED_TICKET_FIELDS = ['status', 'statusUpdatedAt', 'deferredUntil']
+
+const createOfflineTicketStatusChange = async ({ schema, operation, context, existingItem, resolvedData }) => {
+    await storeChangesIfUpdated(
+        pick(schema.fields, ['status']),
```

Comment:
If you update `deferredUntil` then you should pick this field too for these changes to be saved in TicketChange.